### ISSUE TITLE
Error on concurrent requests with enable_static_handler set to true

### ISSUE THREAD

**Issue by ezimuel** [swoole/swoole-src#1810]
1. What did you do? If possible, provide a recipe for reproducing the error.

I executed a simple benchmark to test concurrent requests with `enable_static_handler` set to true.
I used [this example](https://gist.github.com/ezimuel/a2e0ff7308952f2aa946f828a1302a63) (uncomment lines 10-13) and [ab](https://httpd.apache.org/docs/2.4/programs/ab.html) for benchmark, using the command as follows:
```bash
ab -n 100 -c 10 http://localhost:9501/video.mp4
```
where `video.mp4` is a MP4 video of about 700 MB.

2. What did you expect to see?

The ab benchmark fails with error:
```
apr_pollset_poll: The timeout specified has expired (70007)
```

3. What did you see instead?

If you comment lines [10-13](https://gist.github.com/ezimuel/a2e0ff7308952f2aa946f828a1302a63#file-static-php-L10-L13) the script works just fine. Also the performance are very good, a little bit faster than nginx.

4. What version of Swoole are you using (`php --ri swoole`)?

```
swoole support => enabled
Version => 4.0.2
Author => Swoole Group[email: team@swoole.com]
coroutine => enabled
epoll => enabled
eventfd => enabled
timerfd => enabled
signalfd => enabled
cpu affinity => enabled
spinlock => enabled
rwlock => enabled
async http/websocket client => enabled
pcre => enabled
zlib => enabled
mutex_timedlock => enabled
pthread_barrier => enabled
futex => enabled

Directive => Local Value => Master Value
swoole.aio_thread_num => 2 => 2
swoole.display_errors => On => On
swoole.use_namespace => On => On
swoole.use_shortname => On => On
swoole.fast_serialize => Off => Off
swoole.unixsock_buffer_size => 8388608 => 8388608
```

5. What is your machine environment used (including version of kernel & php & gcc) ? 

Ubuntu 18.04
Linux 4.15.0-24-generic
php 7.2.7
gcc 7.3.0

**Comment by ezimuel** [comment]
@twose I just tested again using swoole 4.0.3, same results.

**Comment by matyhtf** [comment]
@ezimuel You can use nginx to handle static files, I will confirm this problem today and fix it.

**Comment by matyhtf** [comment]
This may be caused by http keepalive. 
Try using `ab -k`

```shell
ab -n 100 -c 10 http://localhost:9501/video.mp4
```

**Comment by ezimuel** [comment]
@matyhtf the `-k` option enables the keepalive for [ab](https://httpd.apache.org/docs/2.4/programs/ab.html) (default is not keepalive). Anyway, with keepalive enabled the issue is solved:
```bash
ab -k -n 100 -c 10 http://localhost:9501/video.mp4
```

**Comment by ezimuel** [comment]
@matyhtf I'm testing swoole with [Expressive](https://getexpressive.org/) project and I implemented an adapter, [zend-expressive-swoole](https://github.com/zendframework/zend-expressive-swoole), to support swoole out-of-the-box. I would like to use swoole to serve also web applications written in Expressive, that's why I'm testing also static files (like css, js, etc).

**Comment by twose** [comment]
@ezimuel 
apache bench uses HTTP1.0 and no keep alive, swoole static handle hasn't handled it, I will fix it soon.

**Comment by twose** [comment]
Thanks for your bug report, it fixed in 7c345fa
